code/modules/hydroponics/fermenting_barrel.dm

```diff
@@ -0,0 +1,76 @@
+/obj/structure/fermenting_barrel
+	name = "wooden barrel"
+	desc = "A large wooden barrel. You can ferment fruits and such inside it, or just use it to hold liquid."
+	icon = 'icons/obj/objects.dmi'
+	icon_state = "barrel"
+	density = TRUE
+	anchored = FALSE
+	container_type = DRAINABLE | AMOUNT_VISIBLE
+	var/open = FALSE
+	pressure_resistance = 2 * ONE_ATMOSPHERE
+	max_integrity = 300
+	var/speed_mod = 1 //How fast it distills. Defaults to 100% (1.0). Lower is better.
```

Comment:
I really wish this var had `multiplier` somewhere in it so it was obvious lower is better.

Reply:
Will do.